cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/sidecar/ArtifactLocalizerBase.java

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
new lines in license seems inconsistent with other classes?

Reply:
fixed